Data-Structures/linkedList/src/test/java/linkedList/LibraryTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package linkedList;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test public void testSomeLibraryMethod() {
```

Comment:
Here you can do something like this so you don't have to instantiate so many linkedlists within your tests.
LinkedList link = new LinkedList(); @before public void setUp() { link.insert("Apples", 1); link.insert("Bananas", 2); };

and now, in each one of your tests, you can reference link. `@before` will run before each test and will create that new LinkedList for you, vice having to write the same thing a dozen times over.

Reply:
Thanks, will implement this idea.